content/pages/conference/report.rst

```diff
@@ -0,0 +1,213 @@
+========================================
+Sciwork 2023 Conference Report
+========================================
+
+:date: 2023-12-24 00:00
+:url: conference/report
+:save_as: conference/report.html
+
+First conference for scientific programming
+===============================================================
+
+The first sciwork conference (Hsinchu, 9th-10th December 2023) brought us a lot of fun! 
+The conference focuses on connecting scientific programming with practical applications.
+
+We gathered to discuss subjects for scientific programming and its applications.
+The 15 talks in the technical program included data engineering and science,
+software engineering, array and tensor code development, applications of large language models (LLM),
+natural language processing, graphic user interface (GUI) for numerical analysis, astrology,
+and image processing. It is hard to fully digest the diversified subjects, but it is exciting
+to see how computer code works for everything in science and engineering.
+
+Event Album
+----------------------------------------
+
+If you missed it, check the photos at https://www.flickr.com/photos/sciwork/albums.
+
+
+We will also upload the recordings once they are processed.
+
+.. raw:: html
```

Comment:
The embedded HTML looks a little bit strange on mobile.  It should be OK at the moment.  But it is also good to improve.

Reply:
![image](https://github.com/sciwork/swportal/assets/18013815/1bd2c8c7-18e6-4d50-8ab9-b25d1a07969d)

![image](https://github.com/sciwork/swportal/assets/18013815/7f6250a4-6b26-458c-8412-542c63a73ced)

Reply:
fixed.

Reply:
Your skill works like a magic!  Impressed.